vuepress/docs/next/docs/getting-started/ent-svc.md

```diff
@@ -1,3 +1,57 @@
 ---
 sidebarDepth: 1
 ---
+
+# Auxiliary Services
+
+The following sections discuss auxiliary services and commands.
```

Comment:
Maybe it was listed elsewhere but I think we need to call out that these "auxiliary" services are purely for a local development environment. In Kubernetes these services are provided by the platform (e.g. keycloak or RedHat SSO) or configuration settings can be enabled to point to externally managed services (like mysql or postgresql).  It took me a minute to sort that out.

Reply:
i really don't remember reading that anywhere and it gives some missing context/explanation